templates/certificates/_accomplishment-footer.html

```diff
@@ -0,0 +1,16 @@
+<div class="wrapper-footer">
+
+    <footer class="footer-app" role="contentinfo" id="company-info">
+        <div class="footer-app-copyright">
+            <p class="copy copy-micro">&copy; <script>document.write(new Date().getFullYear())</script> Microsoft Corporation</p>
+        </div>
+        <nav class="footer-app-nav">
+            <ul class="list list-legal">
+                <li class="nav-item">
+                    <a class="action btn btn-small btn-link" href="https://openedx.micrsoft.com/tos">Terms of Service</a>
```

Comment:
Typo in the  href value (missing 'o').  Should be: https://openedx.microsoft.com/tos